### PR TITLE
F51-158 and F51-168

### PR DESCRIPTION
**Pull request by olsonkr**
- Add logic to pretty-submit directive
- Rename pretty-submit to oc-pretty-submit
- Add oc-pretty-submit to missing forms

### CODE REVIEW

File: src/app/productManagement/pricing/templates/createProductPrice.modal.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<form name="createProductPriceModal.form" autocomplete="off" novalidate pretty-submit>
+<form name="createProductPriceModal.form" ng-submit="createProductPriceModal.submit()" autocomplete="off" novalidate oc-pretty-submit>
```
Comment (robertsoniv) [inline src/app/productManagement/pricing/templates/createProductPrice.modal.html]: Did you test out that making these changes didn't mess up the price-breaks directives?

Comment (robertsoniv) [inline src/app/productManagement/pricing/templates/createProductPrice.modal.html]: Did you test out that making these changes didn't mess up the price-breaks directives?

File: src/app/productManagement/pricing/templates/updateProductPrice.modal.html [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<form name="updateProductPriceModal.form">
+<form name="updateProductPriceModal.form" ng-submit="updateProductPriceModal.submit()" oc-pretty-submit>
```
Comment (robertsoniv) [inline src/app/productManagement/pricing/templates/updateProductPrice.modal.html]: Same thing with this one

Comment (olsonkr) [inline src/app/productManagement/pricing/templates/updateProductPrice.modal.html]: It doesn't break the price break directives, but if all the user does is delete a price break, the Save button is still disabled. I'll come up with a resolution for that.

File: src/app/common/directives/oc-pretty-submit/oc-pretty-submit.js [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+                updateForm();
+            });
+
+            scope.$watch(function() { return formCtrl.$dirty; }, function(formDirty) {
+                dirty = formDirty;
+                updateForm();
+            });
+
+            function updateForm() {
+                console.log(element[0].elements);
```
Comment (robertsoniv) [inline src/app/common/directives/oc-pretty-submit/oc-pretty-submit.js]: pls remove console log, pleb

File: src/app/productManagement/pricing/templates/updateProductPrice.modal.html (hunk 2 above)
Comment (olsonkr) [inline src/app/productManagement/pricing/templates/updateProductPrice.modal.html]: That works as expected, yes.

I noticed an issue when all the user does is delete a price break. It did not recognize the model as being updated.

File: src/app/productManagement/pricing/js/ocPriceBreaks.directive.js [hunk 4]
```diff
@@ -89,6 +89,7 @@ function OrderCloudPriceBreaksDirective($filter, ocProductPricing, OrderCloudSDK
                 scope.priceschedule.PriceBreaks.splice(index, 1);
             }
             ocProductPricing.PriceBreaks.FormatQuantities(scope.priceschedule.PriceBreaks);
+            scope.newBreakForm.Price.$setDirty();
```
Comment (robertsoniv) [inline src/app/productManagement/pricing/js/ocPriceBreaks.directive.js]: Perfect!

**Pull request merged**